fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/JcrRdfTools.java

```diff
@@ -414,7 +416,4 @@ private Resource getSkolemizedResource(final IdentifierConverter<Resource, Fedor
         return skolemizedBnodeMap.get(id);
     }
 
-    private static String skolemizedId() {
```

Comment:
Keep this method, as it reduces unnecessary code changes and encapsulates the ".well-known/genid/" location.